.github/workflows/tagged_release.yaml

```
@@ -0,0 +1,103 @@
+name: Create tagged release
+on:
+  push:
+    tags:
+      - v*.*.*
+
+jobs:
+  sonar:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Git repository
+        uses: actions/checkout@v2
+      - name: Set up node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+      - name: Setup sonarqube
+        uses: warchant/setup-sonar-scanner@v3
+      - name: Setup Docker
+        run: |
+          docker pull s1hofmann/nut-ci:latest
+          docker run -it -d --name nut-ci --shm-size 4gb --user $(id -u):$(id -g) -v ${PWD}:${PWD}:rw s1hofmann/nut-ci:latest bash
+      - name: Install
+        run: npm ci
+      - name: Compile
+        run: npm run compile
+      - name: Init e2e test subpackage
+        run: npm --prefix e2e/tests ci
+      - name: Clean coverage report
+        run: npm run coverage:clean
+      - name: Generate coverage report
+        uses: GabrielBB/xvfb-action@v1
+        with:
+          run: npm run coverage -- --coverageDirectory=coverage/unit
+      - name: Run Docker E2E tests
+        run: docker exec nut-ci bash -c "bash $PWD/.build/build.sh ${PWD} 14"
+      - name: Merge coverage reports
+        run: |
+          npm run coverage:merge
+          npm run coverage:merge-report
+      - name: Run sonarqube
+        env:
+          # to get access to secrets.SONAR_TOKEN, provide GITHUB_TOKEN
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: sonar-scanner
+          -Dsonar.login=${{ secrets.SONAR_TOKEN }}
+          -Dsonar.organization=nut-tree
+          -Dsonar.host.url=https://sonarcloud.io/
+          -Dsonar.projectKey=nut-tree:nut.js
+
+  test:
+    needs:
+      - sonar
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, windows-latest, macos-latest ]
+        node: [ 10, 11, 12, 13, 14 ]
```

```suggestion
        node: [ 10, 12, 14 ]
```
same remark as for ci